repo/retention_blob.go

```diff
@@ -0,0 +1,78 @@
+package repo
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+// RetentionBlobID is the identifier of a BLOB that describes BLOB retention
+// settings for the repository.
+const RetentionBlobID = "kopia.retention"
```

Comment:
perhaps `kopia.blobcfg` so we can add more things there?

Reply:
done